### PR TITLE
Namespace client

### PR DESCRIPTION
**Pull request by laoneo**
Namespace client.

### CODE REVIEW

File: libraries/src/Joomla/CMS/Client/ClientWrapper.php [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
 defined('JPATH_PLATFORM') or die;
 
 /**
- * Wrapper class for JClientHelper
+ * Wrapper class for ClientHelper
  *
  * @package     Joomla.Platform
  * @subpackage  Client
  * @since       3.4
+ * @deprecated  4.0 Will b removed without replacemenent
```
Comment (mbabker) [inline libraries/src/Joomla/CMS/Client/ClientWrapper.php]: be, not b ;-)

**Comment by frankmayer** [comment]
@laoneo fix conflict?

**Comment by wilsonge** [comment]
`JLDAP` and `JFTP` need a deprecation version of 4.0 in the classmap file as they are already deprecated :)

**Comment by laoneo** [comment]
Done.

**Pull request merged**